usaspending_api/common/helpers/unit_test_helper.py

```diff
@@ -1,3 +1,21 @@
+from django.core.exceptions import FieldError
+from usaspending_api.download.v2 import download_column_historical_lookups as lookup_mapping
+from usaspending_api.download.lookups import VALUE_MAPPINGS
+
+
 def add_to_mock_objects(mock_obj, mock_models_list):
     for mock_model in mock_models_list:
         mock_obj.add(mock_model)
+
+
+def mappings_test(download_type, sublevel):
+    download_mapping = VALUE_MAPPINGS[download_type]
+    table_name = download_mapping['table_name']
+    table = download_mapping['table']
+
+    try:
+        query_values = lookup_mapping.query_paths[table_name][sublevel].values()
+        table.objects.values(*query_values)
+        assert True
```

Comment:
(for more discussion) should we be asserting in the helper file? or return true/false here and assert the return value in the unit test directly? I'd be inclined to think the latter but was just curious.

Reply:
Latter makes sense, I'll include the change